react/features/base/connection/actions.native.js

```diff
@@ -180,13 +183,13 @@ function _connectionDisconnected(connection, message) {
  *
  * @param {JitsiConnection} connection - The JitsiConnection which was
  * established.
- * @private
  * @returns {{
  *     type: CONNECTION_ESTABLISHED,
  *     connection: JitsiConnection
  * }}
+ * @public
```

Comment:
@lyubomir this is react so the decision is yours. @hristoterezov could you please comply? 
Thanks